kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/storage/ResultProcessors.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2017-2023 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lenses.streamreactor.connect.aws.s3.storage
+
+import io.lenses.streamreactor.connect.aws.s3.model.location.RemoteS3PathLocation
+import software.amazon.awssdk.services.s3.model.S3Object
+
+object ResultProcessors {
+
+  def processObjectsAsFileMeta(
+    bp:        RemoteS3PathLocation,
+    objectSeq: Seq[S3Object],
+  ): Option[ListResponse[FileMetadata]] = {
+    val list = objectSeq
+      .map(s3O => FileMetadata(s3O.key(), s3O.lastModified()))
```

Comment:
Interesting variable name. Any reasons for s30? (there is nothing wrong, just a bit unusual, c/C++ style :)  )

Reply:
s3O is simply "S3 Object" 😆 Maybe the O looks like an 0.

Reply:
that is what it was to me :) thirty